config/settings/development.yml

```diff
@@ -1,3 +1,4 @@
 nndd_url: "http://localhost:9000"
 location_service_url: "http://locations-stg.instedd.org"
+web_session_timeout: 120
```

Comment:
Would you mind increasing this value to 3600, to simplify development and avoid having to log back in frequently?

If you want to develop and test the timeoutable behavior, you can always redefine it in a "development.local.yml" file that will be ignored by git.